README.md

```diff
@@ -1,2 +1,32 @@
-# hw01
-Repository for first CFSS homework assignment
+# ~~hw01~~
+~~Repository for first CFSS homework assignment~~
+
+# Alex Albury Bio
+Hello all (or should I say 'Hello World'?),
+
+I'm an international student from The Bahamas(see islands in yellow below).
+
+<img src="http://i.infopls.com/images/mbahama.gif" width="350" height="350" />
```

Comment:
Good job changing the shape of the image by finding out how to add the extra code!